### PR TITLE
add support link on the navbar

### PR DESCRIPTION
**Pull request by thinkC**
Add Support link to navbar

### CODE REVIEW

File: ui/src/components/navigation.js [hunk 1]
```diff
@@ -34,6 +35,8 @@ export const Navigation = ({
     links
 }) => {
     const currentCloud = get(clouds, currentCloudName);
+    // add this
+    const supportLink = "https://support-link.com";
```
Comment (JohnGarbutt) [inline ui/src/components/navigation.js:39]: This value needs to come from the Azimuth API configuration, just like we have for the documentation please.

Have a look at the links.documentation, and trace that back to here:
https://github.com/azimuth-cloud/azimuth/blob/927026313b7cd62814d20db0d5d9031ac280dc5f/api/azimuth/views.py#L292

And that comes from here (although in your case, you will want this to default to None for the support link case:
https://github.com/azimuth-cloud/azimuth/blob/927026313b7cd62814d20db0d5d9031ac280dc5f/api/azimuth/settings.py#L267

Note the helm chart has the documentation variable here:
https://github.com/azimuth-cloud/azimuth/blob/927026313b7cd62814d20db0d5d9031ac280dc5f/chart/values.yaml#L168
Which is what goes into here:
https://github.com/azimuth-cloud/azimuth/blob/927026313b7cd62814d20db0d5d9031ac280dc5f/chart/files/api/settings/02-cloud-common.yaml#L22

Then to make this usable, we need to inject that via the ansible please:
https://github.com/azimuth-cloud/ansible-collection-azimuth-ops/blob/e85686d5fb03e336797123f7fab7831bc54178cc/roles/azimuth/defaults/main.yml#L87
which goes into the helm chart values here:
https://github.com/azimuth-cloud/ansible-collection-azimuth-ops/blob/e85686d5fb03e336797123f7fab7831bc54178cc/roles/azimuth/defaults/main.yml#L376

File: ui/src/components/navigation.js [hunk 2]
```diff
@@ -89,6 +92,21 @@ export const Navigation = ({
                                 Documentation
                             </Nav.Link>
                         )}
+
+
+                {/* add support link here */}
```
Comment (JohnGarbutt) [inline ui/src/components/navigation.js:97]: I think we can remove these comments and extra blank lines, please align the indentation with the other links above please.

**Review by JohnGarbutt — CHANGES_REQUESTED** [review]
We need to make this link configurable via the ansible and helm chart, then get that passed all the way through please.

It should be similar to the documentation URL, except that support should be off by default please (similar to how cloud metrics is off by default).

File: ui/src/components/navigation.js (hunk 2 above)
Comment (thinkC) [reply ui/src/components/navigation.js:97]: Okay, thanks will make the changes

File: api/azimuth/settings.py [hunk 3]
```diff
@@ -266,5 +266,9 @@ class AzimuthSettings(SettingsObject):
     #: URL for documentation
     DOCUMENTATION_URL = Setting(default = "https://azimuth-cloud.github.io/azimuth-user-docs/")
 
+    #: URL for Support
+    # SUPPORT_URL = Setting(default = None)
+    SUPPORT_URL = Setting(default = "https://confluence.skatelescope.org/display/SRCSC/Azimuth+Support+on+Cambridge+Arcus")
```
Comment (mkjpryor) [inline api/azimuth/settings.py:271]: The default here should be `None`, as in the commented out line above. The SKA Confluence link should be in the Arcus Azimuth config only.

File: ui/src/components/navigation.js [hunk 4]
```diff
@@ -34,6 +35,7 @@ export const Navigation = ({
     links
 }) => {
     const currentCloud = get(clouds, currentCloudName);
+    console.log(links);
```
Comment (mkjpryor) [inline ui/src/components/navigation.js:38]: Please remove debugging statements

File: ui/src/components/navigation.js [hunk 5]
```diff
@@ -89,6 +91,14 @@ export const Navigation = ({
                                 Documentation
                             </Nav.Link>
                         )}
+
```
Comment (mkjpryor) [inline ui/src/components/navigation.js:94]: Remove whitespace for consistency with the rest of the code

File: ui/src/components/navigation.js [hunk 6]
```diff
@@ -89,6 +91,14 @@ export const Navigation = ({
...
+                        {links && links.support && (
+                            <Nav.Link href={links.support} target="_blank" active={false}>
+                                <FontAwesomeIcon icon={faLifeRing} className="me-2" />
+                                Support
+                            </Nav.Link>
+                        )}
+
```
Comment (mkjpryor) [inline ui/src/components/navigation.js:101]: Same here please

**Review by mkjpryor — CHANGES_REQUESTED** [review]
Just a handful of minor changes.

Please also rebase the PR onto the current `master`.

**Review by JohnGarbutt — COMMENTED** [review]
Thank you Tunde. This looks good to me. Will leave @mkjpryor to confirm he is happy now.

**Review by mkjpryor — APPROVED** [review]
LGTM

**Pull request merged**